### DISCUSSION TITLE
Ticket Handling for postponed events

### DISCUSSION THREAD

**Discussion by blutonium** [pretix/pretix#2684] [Q&A]
Hello all,

I have a best practice question.

Basically, it is about the fact that an event with already sold tickets has been postponed. Unfortunately, due to the availability of the artists, there are now two replacement dates, to which the previous ticket buyer should have free access. From now on we want to sell the tickets for both events separately, so we have to create two new events. Now there are two things to manage:

1. for the second replacement event we have to transfer all tickets of the current event to the new event, because in the evening we can only scan for the new event (right?).

2. for the first replacement event it is a bit more difficult. Here the number of seats is limited, so we need to know very precisely who is really coming. My idea for this was to create the existing ticket codes in the new event as vouchers - if possible automated due to the high number. Would this be a practical option or are there better variants?

I'm thankful for any help!

**Comment by raphaelm** [comment]
I don't think there's a great solution for "splitting" events in pretix, unfortunately :(

The cleanest way is probably to cancel all orders and give everyone a gift card and ask them to book new tickets for the new events.

Everything else (like using ticket codes as vouchers) is a not-quite-nice workaround and can be automated eg through the API but will have some rough edges.
